doc/whatsnew.rst

```diff
@@ -62,6 +62,10 @@ v0.2.8.dev
 
 - Add example on principal geodesic analysis (PGA) for SSVEP classification
 
+- Parallelize functions for dataset generation: :func:`pyriemann.datasets.make_gaussian_blobs`
+
+- Fix dispersion when generating datasets: :func:`pyriemann.datasets.sample_gaussian_spd`
```

Comment:
we should split the what's new between Enhancements / New Features and Bug fixes. And we should refer to the PRs on github. See what we use in mne https://github.com/mne-tools/mne-python/blob/main/doc/sphinxext/gh_substitutions.py

Reply:
Yes, I'll push this in a new PR